### ISSUE TITLE
Implement Wallet Home Scroll

### ISSUE THREAD

**Issue by OmarBasem** [status-im/status-react#17086]
Implement the wallet home scroll UI.

Designs: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1348-316521&mode=design&t=qVI9Rq3N7CCZkSRh-0

Before starting this issue let's discuss among the team on ways to approach this as there is other similar implementations in the codebase but (ideally) they can be improved upon and we can use this issue as a means to experiment with this.

**Comment by KaushikKC** [comment]
Can I fix this ?
Would like to get assigned

**Comment by J-Son89** [comment]
Hi @KaushikKC thanks for the interest. Unfortunately we would prefer if this issue is handled by a core contributor as scrolling on these pages is quite advanced and we want to have some discussion to improve upon how we are doing this. If you are looking to start making some contributions ideally you can take a look at some of these:
 https://github.com/status-im/status-mobile/issues?q=is%3Aopen+is%3Aissue+label%3A%22E%3AMobile+Wallet+Foundations+MVP%22
 basically anything with the label `E:Mobile Wallet Foundations MVP`

**Comment by J-Son89** [comment]
Perhaps we can try implementing something like: https://github.com/MayaSaringan/react-native-scroll-context/tree/main/src and using this to avoid uses of global ratoms etc

**Comment by J-Son89** [comment]
@ulisesmac - we should probably createa larger feature epic and add this issue and other related features to that feature epic

i.e implementing the graph
implementing drag down to refresh balances
implementing the home page scroll

**Comment by ulisesmac** [comment]
Yeah :+1: 

it'd have its own epic or it'll be part of another epic?

**Comment by ulisesmac** [comment]
In previous discussions we agreed to remove the graph and the complex scroll. So this issue is now blocked until we decide to return these features.
